modules/cluster-logging-exported-fields-top-level-fields.adoc

```diff
@@ -0,0 +1,152 @@
+[id="cluster-logging-exported-fields-top-level-fields_{context}"]
+== Top-level fields
+
+// Normally, the preceding title would be an H1 prefixed with an `=`. However, because the following content is auto-generated at https://github.com/ViaQ/documentation/blob/main/src/data_model/public/top-level.part.adoc and pasted here, it is more efficient to use it as-is with no modifications. Therefore, to "realign" the content, I am going to prefix the title with `==` and use `include::modules/cluster-logging-exported-fields-top-level-fields.adoc[leveloffset=-1]` in the assembly file.
+
+// DO NOT MODIFY THE FOLLOWING CONTENT. Instead, update https://github.com/ViaQ/documentation/blob/main/src/data_model/model/top-level.yaml and run `make` as instructed here: https://github.com/ViaQ/documentation
+
+The top level fields may be present in every record.
+
+
+
+=== message
+
+Original log entry text, UTF-8 encoded
+
+[horizontal]
+Data type:: text
+
+Example value:: `HAPPY`
+
+
+
+
+
+=== @timestamp
+
+A UTC value that marks when the log payload was created or when the log payload was first collected if the creation time is not known. This value is the log processing pipeline's “best effort” determination of when the log payload was generated. The “@” prefix denotes a field that is reserved for a particular use. By default, most tools look for “@timestamp” with ElasticSearch.
```

Comment:
Instead of best effort, can you use the word `estimate`?

Reply:
Thanks for raising this question. I restructured the first phrase to be clearer and removed the "best effort" phrase as redundant.